include/deal.II/algorithms/operator.h

```diff
@@ -106,7 +106,7 @@ namespace Algorithms
   template <typename VectorType>
   class OutputOperator : public Subscriptor
   {
-    OutputOperator(const OutputOperator<VectorType> &);
+    OutputOperator(const OutputOperator<VectorType> &) = delete;
```

Comment:
If you do this, then you might as well put it into the `public` section where we usually have constructors.